bootstrap/src/main/kotlin/jp/co/soramitsu/notary/bootstrap/dto/Model.kt

```diff
@@ -0,0 +1,90 @@
+package jp.co.soramitsu.notary.bootstrap.dto
+
+import jp.co.soramitsu.iroha.java.TransactionBuilder
+import jp.co.soramitsu.notary.bootstrap.genesis.getIrohaPublicKeyFromHexString
+import java.security.KeyPair
+import javax.xml.bind.DatatypeConverter
+
+private interface DtoFactory<out T> {
+    fun getDTO(): T
+}
+
+open class Conflicatable(var errorCode: String? = null, var message: String? = null)
+
+data class BlockchainCreds(
+    val private: String? = null,
+    val public: String? = null,
+    val address: String? = null
+)
+
+data class IrohaAccountDto(
+    val title: String = "",
```

Comment:
We use terminology `account_name` in Iroha docs for it https://iroha.readthedocs.io/en/latest/api/commands.html#create-account

Reply:
fixed to name